airflow/executors/base_executor.py

```diff
@@ -65,11 +65,13 @@ class BaseExecutor(LoggingMixin):
     """
 
     supports_ad_hoc_ti_run: bool = False
+    supports_sentry: bool = False
 
     job_id: None | int | str = None
     callback_sink: BaseCallbackSink | None = None
 
     is_local: bool = False
+    is_picklable: bool = False
```

Comment:
I think we want the default to be True, no? The previous code assumed all executors were picklable unless it was present in the list of `UNPICKLEABLE_EXECUTORS`. So any executor that goes through that code should be assumed to be picklable unless it says otherwise. WDYT?

Reply:
I agree that this makes more sense. For some reason I went with the opposite 🙃